src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/summaryjavadoc/InputSummaryJavadocInlineReturn.java

```diff
@@ -0,0 +1,45 @@
+/*
+SummaryJavadoc
+violateExecutionOnNonTightHtml = (default)false
+forbiddenSummaryFragments = (default)^$
+period = (default).
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.javadoc.summaryjavadoc;
+
+class InputSummaryJavadocInlineReturn {
+    /**
+     * {@return a zero, note that no period is ok as it is set by javadoc tool}
+     */
+    private int returnFoo1() // ok
+    {
+        return 0;
+    }
+
+    /**
+     * {@return a {@code 0}}
+     */
+    private int returnFoo2() // ok
+    {
+        return 0;
+    }
+
+    // violation below
+    /**
+     * {@return}
+     */
+    int returnNothing() {
+        return 0;
+    }
+
+    /**
+     * {@return nothing, this is a field}
+     */
+    private static final byte NOT_A_METHOD = 0; // ok
```

Comment:
Sorry if I missed it, but can you point to where we analyzed how the javadoc tool behaves with this or the class one?

Reply:
@octylFractal , please repost your comment to issue.

PR in highly unlikely to be reviewed later on, but issues are good source of our decisions to review through time. Javadoc tool might change behavior and will be puzzleed why we did/didn't do something.

Reply:
Done.